theme/macarons2.js

```diff
@@ -186,15 +183,15 @@
                     color: '#1b1b1b',
                     lineStyle: { color: '#1b1b1b' }
                     // textStyle: null      // 默认使用全局文本样式，详见TEXTSTYLE
-                },
-                labelLine: {
-                    show: true,
-                    length: 20,
-                    lineStyle: {
-                        // color: 各异,
-                        width: 1,
-                        type: 'solid'
-                    }
+                }
```

Comment:
The `label` block should be moved to the outside as well.

Reply:
Okay, I changed it again.😁